packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -97,87 +116,85 @@ const metadata: Meta<WaferMapArgs> = {
-            options: ['set1', 'set2'],
+        colorsScale: {
+            description: `Represents the color spectrum which shows the status of the dies on the wafer.
+                <details>
+                    <summary>Usage details</summary>
+                    The \`colorsScale\` element is a public property. As such, it is not available as an attribute, however it can be read or set on the corresponding \`WaferMap\` DOM element.
+                </details>
+                `,
+            options: ['set1'],
             control: {
                 type: 'radio',
                 labels: {
-                    set1: 'Set 1',
-                    set2: 'Set 2'
+                    set1: 'Scale 1'
                 }
             },
-            defaultValue: 'set1'
+            defaultValue: 'set1',
+            mapping: {
+                set1: waferMapColorsScaleSets[0]
+            }
         },
-        quadrant: {
+        colorsScaleMode: {
+            name: 'color-scale-mode',
             description:
-                'Represents the orientation of the dies on the wafer map',
-            options: Object.values(WaferMapQuadrant),
+                'Enum value that determines if the color scale represents continuous gradient values (linear), or is set categorically (ordinal).',
+            options: Object.values(WaferMapColorsScaleMode),
             control: {
                 type: 'radio',
                 labels: {
-                    [WaferMapQuadrant.bottomLeft]: 'bottom-left',
-                    [WaferMapQuadrant.bottomRight]: 'bottom-right',
-                    [WaferMapQuadrant.topLeft]: 'top-left',
-                    [WaferMapQuadrant.topRight]: 'top-right'
+                    [WaferMapColorsScaleMode.linear]: 'Linear',
+                    [WaferMapColorsScaleMode.ordinal]: 'Ordinal'
                 }
             }
         },
-        orientation: {
-            description: 'Notch orientation',
-            options: Object.values(WaferMapOrientation),
+        dies: {
+            description: `Represents the input data, an array of \`WaferMapDie\`, which will be renedered by the wafer map
+                <details>
+                    <summary>Usage details</summary>
+                    The \`dies\` element is a public property. As such, it is not available as an attribute, however it can be read or set on the corresponding \`WaferMap\` DOM element.
+                </details>
+                `,
+            options: ['set1', 'set2'],
             control: {
                 type: 'radio',
                 labels: {
-                    [WaferMapOrientation.left]: 'left',
-                    [WaferMapOrientation.top]: 'top',
-                    [WaferMapOrientation.right]: 'right',
-                    [WaferMapOrientation.bottom]: 'bottom'
+                    set1: 'Set 1',
+                    set2: 'Set 2'
```

Comment:
Is there a more descriptive description for the sets? As a user looking at the storybook I don't know that to expect when choosing a set. If they had names like "lots of failure", "high values", "large variance", "small variance", etc. whatever makes the set of data interesting.

Reply:
Currently the values contained within the sets are rather random - they do not follow any specific pattern, only for the user to see a change if he/she picks a different option. I agree, however, to go for more descriptive names once we come up with some more concrete scenarios we want to display.